typescript/sdk/src/providers/SmartProvider/SmartProvider.test.ts

```diff
@@ -181,11 +179,10 @@ describe('SmartProvider', () => {
       });
 
       it('Handles parallel requests', async () => {
-        const result1Promise = provider.getLogs({
-          address: WETH_CONTRACT,
-          topics: [WETH_TRANSFER_TOPIC0],
-          fromBlock: MIN_BLOCK_NUM,
-          toBlock: MIN_BLOCK_NUM + 100,
+        const result1Promise = provider.call({
```

Comment:
why change it from getLogs to call?

Reply:
Oh just to simplify this test since there's not a static min block number anymore. The nature of the call isn't important here, just that 3 run in parallel.